### PR TITLE
Added some Ka'het hails

### PR DESCRIPTION
**Pull request by beccabunny**
**Content (Artwork / Missions / Jobs)**

## Summary
This PR adds some hails to the Ka'het, to serve as a base for the future developement of the faction - in fact, currently they can't be seen by the player, since he has no way to get the "language: Ka'het" yet. About the hails themselves, and why they exist: while you can't really "speak" with a Het, its shell can send you informations about its state. A Het is basically a slug with lasers and a giant exoskeleton, and while you can't talk with the slug you can make it so the exoskeleton outputs its state.
*A hostile hail:*
Screenshot (1315)
*A friendly disabled hail:*
Screenshot (1319)

### CODE REVIEW

**Comment by downRightCoder256** [comment]
isn’t by game lore disabled het a dead het? why is it asking for assistance?

**Comment by beccabunny** [comment]
Because as i said in the opening post, you aren't talking with the Het, but with the exoskeleton. Since you can't prevent the player from assisting disabled any disabled ship, it seems worth to at least do a hail for it

**Comment by downRightCoder256** [comment]
no i mean like, instead of saying " life support systems: partially online" say something like "life support systems: offline" or in some other way indicate that the actual Het is dead.

**Comment by beccabunny** [comment]
The systems described in the hail are random, but yea, I can change the state of the life support systems to offline (or perhaps unavailable)

**Comment by beccabunny** [comment]
immagine
immagine
immagine
Added two more hails, one with both a friendly and hostile versions and one only friendly

**Comment by beccabunny** [comment]
Screenshot (1328)
Screenshot (1327)
New friendly hails

**Comment by downRightCoder256** [comment]
each one better than the previous. keep it up, Becca !

**Comment by downRightCoder256** [comment]
should open an issue as an reminder to add that information to the ka’het storyline

File: data/ka'het/ka'het.txt [hunk 1]
```diff
@@ -73,6 +73,455 @@ phrase "ka'mar"
...
+		"2"
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		" years, "
+	word
+		""
+		"1 month"
```
Comment (Amazinite) [inline data/ka'het/ka'het.txt:319]: Would months be an appropriate unit of time for an alien to use? Perhaps if these are said to just be translated, but  having the same 12 months in a year that we do is what's really odd.

File: data/ka'het/ka'het.txt [hunk 2]
```diff
@@ -73,6 +73,455 @@ phrase "ka'mar"
...
+
+phrase "friendly ka'het"
+	word
+		"Current"
+		"Selected"
+		"Chosen"
+		"Main"
+		"Primary"
+	word
+		" target: <ship>. Government: "
```
Comment (Amazinite) [inline data/ka'het/ka'het.txt:210]: Does this actually work?

File: data/ka'het/ka'het.txt [hunk 3]
```diff
@@ -73,6 +73,455 @@ phrase "ka'mar"
...
+
+phrase "friendly ka'het"
+	word
+		"Current objective: systems "
+	word
+		"patrol"
+		"garrison"
+		"vigilance"
+	word
+		". Analyzing system <system>. "
```
Comment (Amazinite) [inline data/ka'het/ka'het.txt:353]: Does this work?

File: data/ka'het/ka'het.txt (hunk 2 above)
Comment (beccabunny) [reply data/ka'het/ka'het.txt:210]: Absolutely, see the comments for screenshots

File: data/ka'het/ka'het.txt (hunk 3 above)
Comment (beccabunny) [reply data/ka'het/ka'het.txt:353]: Yes, I don't know if I shared screenshots with this one on github, but when I added this I tested it and it worked

File: data/ka'het/ka'het.txt (hunk 1 above)
Comment (beccabunny) [reply data/ka'het/ka'het.txt:319]: changed to seasons

**Pull request merged**